next/docs/iota-sdk/docs/how-tos/exchange-guide.mdx

```diff
@@ -0,0 +1,260 @@
+---
+description: Easily integrate with your exchange, custody solution, or product using the IOTA SDK.
+sidebar_label: Integrate IOTA SDK With Your Exchange
+keywords:
+  - integrate
+  - exchange
+  - account model
+  - addresses
+  - IOTA SDK
+  - setup
+  - NodeJS
+  - explanation
+---
+
+import WarningUnlockConditions from '../_admonitions/_warning-unlock-conditions.md';
+
+# Stardust Exchange Integration Guide
+
+:::note
+
+You can easily integrate the IOTA SDK in your exchange, custody solution, or product.
+
+:::
+
+:::warning
+
+The stardust update allows for complex tokenization schemes that can lead to monetary losses if misused.
+Transaction outputs may have multiple unlocking conditions which may require returning some or all of the amount, which could expire if not claimed in time, or which may not be unlockable for a very long time. You can lose money if you do not check the unlock conditions before accepting a deposit!
+
+:::
+
+## Integration Guide
+
+This guide explains how to integrate the IOTA SDK into your exchange.
+
+Features of the IOTA SDK:
+
+- Secure seed management.
+- Account management with multiple accounts and multiple addresses.
+- Confirmation monitoring.
+- Deposit address monitoring.
+- Backup and restore functionality.
+
+### How Does it Work?
```

Comment:
```suggestion
### How Does It Work?
```

Reply:
@Dr-Electron Electron what's the policy for these? I think small connecting words are not capitalized?

Reply:
I checked with this tool: https://capitalizemytitle.com/style/APA/

Reply:
https://wiki.iota.org/community/contribute-to-wiki/reference/style/formatting/#headings

So Alex is correct